src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandler.java

```diff
@@ -171,16 +171,13 @@ protected PreSubmitCallbackResponse<SscsCaseData> issueDocument(Callback<SscsCas
     }
 
     protected String getDocumentTypeLabel(SscsCaseData caseData, DocumentType documentType, String documentTypeLabel, boolean isPostHearingsEnabled) {
-
         String embeddedDocumentTypeLabel = (FINAL_DECISION_NOTICE.equals(documentType) ? "Decision Notice" : documentTypeLabel);
 
         if (isPostHearingsEnabled) {
-            PostHearing postHearing = caseData.getPostHearing();
-            if (nonNull(postHearing.getSetAside().getAction())) {
-                CcdCallbackMap action = postHearing.getSetAside().getAction();
-                boolean isGrantOrRefuseSetAsideAction = action.toString().equals(SetAsideActions.GRANT.getCcdDefinition())
-                        || action.toString().equals(SetAsideActions.REFUSE.getCcdDefinition());
-                embeddedDocumentTypeLabel =  isGrantOrRefuseSetAsideAction ? "Set Aside Decision Notice" : embeddedDocumentTypeLabel;
+            PostHearingReviewType postHearingReviewType = caseData.getPostHearing().getReviewType();
+
+            if (nonNull(postHearingReviewType)) {
+                return postHearingReviewType.getDescriptionEn() + " Decision Notice";
```

Comment:
What is the reason for deleting this block of code? Has it been made redundant by getDescriptionEn?

Reply:
yeah so before it was only changing it for set aside, we now want to change it for every journey